packages/assets-controllers/src/TokensController.ts

```diff
@@ -194,20 +196,20 @@ export class TokensController extends BaseController<
    * Tokens controller options
    * @param options - Constructor options.
    * @param options.chainId - The chain ID of the current network.
-   * @param options.selectedAddress - Vault selected address
```

Comment:
Do you know why does it say `vault`? Does it make sense?

Reply:
Param has been removed. It should have been Account of instead of vault.